lib/kredis/types/unique_list.rb

```diff
@@ -16,7 +16,7 @@ def append(elements)
     multi do
       remove elements
       super
-      ltrim (limit - 1), -1 if limit
+      ltrim -limit, -1 if limit
```

Comment:
Interesting that going from the tail end isn't zero-indexed, e.g. the base is `-1` and not `-0`.

Reply:
I think it matches Ruby with ranges:

```ruby
irb(main):001:0> [1,2,3,4][-3..-1]
=> [2, 3, 4]
```